tests/acceptance/features/webUIPreview/mediaPreview.feature

```diff
@@ -55,106 +27,3 @@ Feature: display image in preview app on the webUI
       | resource-one    | resource-two    |
       | test_video0.mp4 | test_video1.mp4 |
       | testavatar.jpg  | testavatar.png  |
-
-
-  Scenario: downloading media resource is possible
```

Comment:
do we have e2e test for this case?

Reply:
We don't have e2e test for downloading resource from appbar while it is being previewed. I'll add this step.